libraries/AP_Arming/AP_Arming.h

```diff
@@ -303,6 +303,9 @@ class AP_Arming {
 
     uint32_t last_prearm_display_ms;  // last time we send statustexts for prearm failures
     bool running_arming_checks;  // true if the arming checks currently being performed are being done because the vehicle is trying to arm the vehicle
+
+    bool last_prearm_checks_result;
```

Comment:
Let's put some comments beside these variables to be consistent with the others above.

Reply:
good idea. Done.